chore(react-scheduler-demos): add the Overview demo (#2361)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/demos/featured/overview.md

# React Scheduler Overview

> The DevExtreme Reactive Scheduler is a React component for scheduling and displaying appointments. Its features include:

Reviewer: ```suggestion
The DevExtreme Reactive Scheduler enables you to schedule and display appointments. Its features include:
```

Reviewer 2: ```suggestion
The DevExtreme Reactive Scheduler enables its users to schedule and display appointments. Its features include:
```
